repo2docker/buildpacks/conda/freeze.py

```diff
@@ -12,7 +12,7 @@
 import os
 import pathlib
 from argparse import ArgumentParser
-from datetime import datetime
+from datetime import UTC, datetime
```

Comment:
This is `datetime.timezone.UTC`. And `timezone.UTC` is a class attribute, so not directly importable which means you have to do something like

```suggestion
from datetime import datetime, timezone
UTC = timezone.UTC
```

if you want UTC as a top-level variable.